backend/dal/Services/Concrete/BuildingService.cs

```diff
@@ -167,8 +167,8 @@ public Building Add(Building building)
 
             // A building should have a unique name within the parcel it is located on.
             building.Parcels.ForEach(pb => this.Context.ThrowIfNotUnique(pb.Parcel, building));
-            // SRES users allowed to overwrite
-            if (!this.User.HasPermission(Permissions.AdminProperties))
+            // SRES users allowed to overwrite, or if user belongs to parent ministry
+            if (!this.User.HasPermission(Permissions.AdminProperties) && agency.ParentId != null)
```

Comment:
Want to confirm this isn't a security issue, need to allow parent ministries to override their user agency to submit on behalf of another

Reply:
If I'm reading this correctly, you allow a **SRES** user or a user who is assigned to a child agency to modify the building's owning agency.  This would be invalid for two reasons.
1) Any user belonging to a parent agency can change the agency of the property **as long as it is an agency they are assigned**
2) A user with `admin-properties` claim can change the agency to anything

Reply:
This is saying it will override it to the user's agency if the user is not an admin and their agency is not a parent agency, originally it always overrided to the user's agency but a while ago I added the check to only do so if they are not an admin based off an older story to allow SRES to update. Will update the documentation so it's less confusing